resources/definitions/os_discovery/fortigate.yaml

```diff
@@ -10,7 +10,20 @@ modules:
     os:
         version: FORTINET-FORTIGATE-MIB::fgSysVersion.0
         serial: ENTITY-MIB::entPhysicalSerialNum.1
-        hardware: ENTITY-MIB::entPhysicalModelName.1
+        hardware:
+                    - SNMPv2-MIB::sysObjectID.0
```

Comment:
This causes an extra snmpget, do you think the yaml is going to work? Might just be easier to do this in PHP.

Maybe look at Junose?

Reply:
the idea is not to avoid php ?
I think, we can remove ENTITY-MIB::entPhysicalModelName.1 it is no longer used

Reply:
i can look to use entPhysicalModelName and rename to have better display name...